jujugui/static/gui/src/app/components/deployment-bar/deployment-bar.js

```diff
@@ -77,12 +77,35 @@ YUI.add('deployment-bar', function() {
       Export the env when the button is clicked.
 
       @method _handleExport
-      @returns {String} the label for the deploy button
     */
     _handleExport: function() {
       this.props.exportEnvironmentFile();
     },
 
+    /**
+      Open a file picker when the button is clicked.
+
+      @method _handleImport
+    */
+    _handleImportClick: function() {
+      var input = this.refs['file-input'];
+      if (input) {
+        input.click();
+      }
+    },
+
+    /**
+      Open a file picker when the button is clicked.
+
+      @method _handleImport
+    */
+    _handleImportFile: function() {
+      var input = this.refs['file-input'];
+      app._renderDragOverNotification(false);
```

Comment:
This method needs to be passed in as a prop, you aren't 'allowed' to access things not passed in.